cmd/adsysd/integration_tests/testdata/TestServiceStatus/golden/ubuntu_pro_subscription_is_not_active

```diff
@@ -4,7 +4,12 @@ Connected users:
   user2@example.com, updated on DDD MON D HH:MM
 Next Refresh: Tue May 25 14:55
 
-Ubuntu Pro subscription is not active on this machine. Some policies won't be applied.
+Ubuntu Pro subscription is not active on this machine. Rules belonging to the following policy types will not be applied:
+  - privilege
```

Comment:
I would expect ascii order there, no? (I think you have the initial slice in the order of the manager of rules to apply, which makes sense, so maybe we should only sort them in status?)